pkg/resources/common/formatter_test.go

```diff
@@ -935,6 +936,79 @@ func Test_formatterLinks(t *testing.T) {
 				"view":    "/api/v1/namespaces/example-ns/pods/example-pod",
 			},
 		},
+		{
+			name:    "patch permissions",
+			hasUser: true,
+			permissions: &permissions{
+				hasPatch: true,
+			},
+			schema: &types.APISchema{
+				Schema: &schemas.Schema{
+					ID: "example",
+					Attributes: map[string]interface{}{
+						"group":    "apps",
+						"version":  "v1",
+						"resource": "deployments",
+					},
+				},
+			},
+			apiObject: types.APIObject{
+				ID: "example",
+				Object: &v1.Pod{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "example-deployment",
+						Namespace: "example-ns",
+					},
+				},
+			},
+			currentLinks: map[string]string{
+				"default": "defaultVal",
+				"view":    "/apis/apps/v1/namespaces/example-ns/deployments/example-deployment",
```

Comment:
`"view"` should be `"patch"` , assuming you delete the two lines from `formatter.go`

Reply:
test updated to reflect the new world order